README.md

```diff
@@ -1,2 +1,52 @@
-# wallet-connect-sdk
-A Wallet Connect SDK for ecosystem tools
+# Wallet Connect SDK
+A Coz's [Wallet Connect](https://walletconnect.org/) 2.0 SDK for ecosystem tools.
+
+**Wallet Connect integration is a secure solution when creating a Decentralized Application that needs user's Account
+authentication. This SDK comes in handy to help on such integration.** 
+
+- [Why do I need Wallet Connect in my dApp?](#why-do-i-need-wallet-connect-in-my-dapp)
+- [Usage example video](#usage-example-video)
+- [How does it work?](#how-does-it-work)
+- [Installation and Setup Guide](#installation-and-setup-guide)
+- [Using with React.js](packages/wallet-connect-sdk-react/README.md)
+- [Using without React.js](packages/wallet-connect-sdk-core/README.md)
+- [How to test my dApp?](#how-to-test-my-dapp)
+
+## Why do I need Wallet Connect in my dApp?
+Almost every dApp needs user's authentication to send a signed transaction to the blockchain, the most common scenery is
+when the client-side application needs to call a SmartContract method that needs the user's Account.
+
+Without a solution like Wallet Connect's integration the user would need to trust his privateKey to the dApp, which is
+a huge security issue, the dApp could simply use it maliciously stealing the user's funds or signing something
+not approved by the user.
+
+Wallet Connect is the most famous solution to avoid this problem, and it's already used by many dApps and users.
```

Comment:
Wallet Connect is the most famous solution to tackle this issue, and it's already used by many dApps and users.

[Não entendi muito bem. Está se referindo ao Wallet Connect original ou ao nosso? E ainda, o nosso Wallet Connect é só pro Neo? Fora do Neo qual a diferença entre ele e o original?]

Reply:
a reminder for myself: make it clear that the wallet connect is very used and we are wrapping it up on this library that facilitates it

Reply:
Fixed, here I've put the WalletConnect website link to be more clear that I am talking about the original Wallet Connect